frontend/src/components/Shared/CrossingListItem/CrossingCommunityList/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './CrossingCommunityList.css'
+
+export default function CrossingCommunityList({ crossing }) {
+  return (
+    <div className="CrossingCommunityList">
+      {crossing.communities.nodes.map(community => (
+        <a href="/" key={community.id}>
+          {/* TODO: Link to community */}
```

Comment:
Not sure what exactly clicking a community is supposed to do. We should create an issue to define this behavior (Public and Dashboard)